### PR TITLE
Added implementers-guide template

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
Hi Ron,
+ The structure of the template is based on "recommendations" template.
+ I'm not sure which text part should be considered as boilerplate. So, in some cases, I included commented text and others normal text.

Cheers

### CODE REVIEW

File: implementers-guide/sections/00-summary.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+== Summary
+This document is ...
+
+This revision contains all updates submitted up to and including those at Study Group YY meeting in {Month Year}.
+
+This document was approved by ITU-T Study Group YY on {Day Month Year} and {obsoletes | complements} the earlier version of this implementers' guide approved on {Day Month Year}.
+
+
+
+== Change log
```
Comment (ronaldtse) [inline implementers-guide/sections/00-summary.adoc:11]: Change all "tabs" to "spaces"

File: implementers-guide/sections/00-summary.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+[abstract]
+== Summary
+This document is ...
+
+This revision contains all updates submitted up to and including those at Study Group YY meeting in {Month Year}.
+
+This document was approved by ITU-T Study Group YY on {Day Month Year} and {obsoletes | complements} the earlier version of this implementers' guide approved on {Day Month Year}.
```
Comment (ronaldtse) [inline implementers-guide/sections/00-summary.adoc]: Do the `{Day Month Year}` work in ADoc? They might be considered to be document attributes?

File: implementers-guide/sections/02-introduction.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+
+== Introduction
+
+This implementers' guide is a compilation of reported defects for all versions of the {Rec # | Series} of Recommendations. In this edition of the guide, reported defects identified as of {yyyy-mm} are given for:
+
+
+// list of recommendations to which the text applies
```
Comment (ronaldtse) [inline implementers-guide/sections/02-introduction.adoc]: Can we make this into actual text so it renders for the user?

File: implementers-guide/sections/05-nomenclature.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+
+== Nomenclature
+
+////
```
Comment (ronaldtse) [inline implementers-guide/sections/05-nomenclature.adoc]: I think we should give an actual example. Can you find one on this page? https://www.itu.int/en/ITU-T/studygroups/com15/Pages/impguides.aspx

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: implementers-guide/sections/00-summary.adoc (hunk 2 above)
Comment (manuelfuenmayor) [reply implementers-guide/sections/00-summary.adoc]: Yes, it renders properly. I shifted the text style to italic.

File: implementers-guide/sections/02-introduction.adoc (hunk 3 above)
Comment (manuelfuenmayor) [reply implementers-guide/sections/02-introduction.adoc]: Yes, change applied.

File: implementers-guide/sections/05-nomenclature.adoc (hunk 4 above)
Comment (manuelfuenmayor) [reply implementers-guide/sections/05-nomenclature.adoc]: I couldn't find a nomeclature example in the provided link. So, I put the example shown in the original template doc.

File: implementers-guide/sections/00-summary.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply implementers-guide/sections/00-summary.adoc:11]: Mmm... I don't see any tab indentation. Only empty lines.

**Review by ronaldtse — APPROVED** [review]

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**